content/_data/changelogs/weekly.yml

```diff
@@ -12362,6 +12363,59 @@
   # pull: 5671 (PR title: Mark Resource Domain Recommendation monitor as security)
   # pull: 5676 (PR title: Bump mockito-core from 3.11.2 to 3.12.1)
 
+  - version: '2.309'
+    date: 2021-08-30
+    changes:
+      - type: TODO
+        category: developer
+        pull: 5685
+        authors:
+          - dependabot[bot]
+          - basil
+        pr_title: Bump XStream from 1.4.17 to 1.4.18
+        message: |-
+          (No proposed changelog)
```

Comment:
I suspect this wasn’t found because the full PR template wasn’t restored. I think it looks for the next section for terminating the changelog